src/video/emscripten/SDL_emscriptenmouse.c

```diff
@@ -206,7 +230,7 @@ Emscripten_ShowCursor(SDL_Cursor* cursor)
             curdata = (Emscripten_CursorData *) cursor->driverdata;
 
             if(curdata->system_cursor) {
-                EM_ASM_INT({
+                MAIN_THREAD_EM_ASM({
```

Comment:
Since you've changed this to the non `_INT` version (which is good, it never needed to return anything), drop the `return 0` below.

(Hmm, need to rework this to not hardcode `Module['canvas']` at some point...)